src/main/java/org/akhq/middlewares/CustomHttpResponseHeadersFilter.java

```diff
@@ -0,0 +1,75 @@
+package org.akhq.middlewares;
+
+
+import io.micronaut.context.ApplicationContext;
+import io.micronaut.core.async.publisher.Publishers;
+import io.micronaut.http.HttpRequest;
+import io.micronaut.http.MutableHttpResponse;
+import io.micronaut.http.annotation.Filter;
+import io.micronaut.http.filter.HttpServerFilter;
+import io.micronaut.http.filter.ServerFilterChain;
+import org.reactivestreams.Publisher;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.inject.Inject;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+@Filter("/**")
+public class CustomHttpResponseHeadersFilter implements HttpServerFilter {
+
+    private static final Logger LOG = LoggerFactory.getLogger(CustomHttpResponseHeadersFilter.class);
+    public static final String PROPERTY_CUSTOM_RESPONSE_HEADERS = "akhq.http.headers.custom-response-headers";

```

Comment:
```suggestion
    public static final Map<String, String> PROPERTY_CUSTOM_RESPONSE_HEADERS = "akhq.server.custom-response-headers";
```
Let users provide the headers as Map directly and change the variable name

Reply:
Well that didn't work in my unit tests.

Reply:
Maybe you need to create a true configuration for that one like this [one](https://github.com/tchiotludo/akhq/blob/dev/src/main/java/org/akhq/configs/Connection.java#L30)

Reply:
Well, we cannot assign a String to a Map as you suggested: ```java
public static final Map<String, String> PROPERTY_CUSTOM_RESPONSE_HEADERS = "akhq.server.custom-response-headers";
```

Reply:
Yes I know I want people to fill it like that: 

```yaml 

akhq:
  server:
    custom-response-headers:
      customheaders: values
      customheaders2: values2

```

Reply:
good catch :+1: 
```yaml 
akhq:
  server:
    custom-response-headers:
     - key: "custom-header-1"
       value: "values1a values1b"
```

Reply:
Configuration is now in class org.akhq.configs.Server. Example:
```yaml
akhq:
  server:
    customHttpResponseHeaders:
      - name: "Content-Security-Policy"
        value: "default-src 'none'; frame-src 'self'; script-src 'self'; connect-src 'self'; img-src 'self'; style-src 'self'; frame-ancestors 'self'; form-action 'self'; upgrade-insecure-requests"
      - name: "X-Permitted-Cross-Domain-Policies"
        value: "none"
```